commands/certmanager/delete.go

```diff
@@ -74,12 +74,18 @@ func CmdDelete(c *core.CommandConfig) error {
 			}
 		}
 	} else {
+		fmt.Fprintf(c.Command.Command.ErrOrStderr(), jsontabwriter.GenerateVerboseOutput("Deleting Certificate..."))
+
 		id, err := c.Command.Command.Flags().GetString(FlagCertId)
 		if err != nil {
 			return err
 		}
+		name, error := c.Command.Command.Flags().GetString(FlagCertName)
```

Comment:
I'm sorry if I'm missing something, but this flag doesnt exist, and in any case, we need to retrieve the actual properties of the resource (so, let's make an API call via SDK and retrieve the name from there)